lib/atom-cfn-lint.js

```diff
@@ -66,17 +130,28 @@ export default {
       lintsOnChange: false,
       lint: (activeEditor) => {
         // setup variables
+        if (!atom.workspace.isTextEditor(activeEditor)) {
+          // If we somehow get fed an invalid TextEditor just immediately return
+          return null
+        }
 
-        const helpers = require('atom-linter');
         const file = activeEditor.getPath();
+        if (!file) {
+          // The editor currently has no path, we can't report messages back to
+          // Linter so just return null
+          return null
+        }
+
+        loadDeps()
```

Comment:
Most of this file seems to use `;`, however the new code is missing it, you might want to configure a linter to ensure consistent style 😉.

Reply:
done.  Much cleaner.